app/main.py

```diff
@@ -2,9 +2,14 @@
 
 from . import api
 from .database import initialize_db
+from .config import config, Environment

```

Comment:
```python
from .config import config, Environment # <--- Should be first since c < d in lexicographical order.
from .database import initialize_db
```